pkg/koordlet/metriccache/metric_types.go

```diff
@@ -54,6 +54,27 @@ const (
 
 	PodMetricCPUThrottled       MetricKind = "pod_cpu_throttled"
 	ContainerMetricCPUThrottled MetricKind = "container_cpu_throttled"
+
+	// CPI
+	ContainerMetricCPICycle       MetricKind = "container_cpi_cycle"
+	ContainerMetricCPIInstruction MetricKind = "container_cpi_instruction"
+
+	// PSI
```

Comment:
then we only need ContainerMetricPSI and PodMetricPSI here, and some/full, cpu/mem/io, 10/60/300 are all MetricProperty